content/machine-learning/lss.en.Rmarkdown

```diff
@@ -29,7 +29,7 @@ corp_news <- download("data_corpus_guardian")
 corp_news <- readRDS("../data/data_corpus_guardian.rds")
 ```
 
-We segment news articles into sentences in the corpus to accurately estimate semantic proximity between words. We also use the [Marimo](https://github.com/koheiw/marimo) stopwords list (`source = "marimo"`) to remove words commonly used in news reports.
+We will segment news articles into sentences in the corpus to accurately estimate semantic proximity between words. We will also use the [Marimo](https://github.com/koheiw/marimo) stopwords list (`source = "marimo"`) to remove words commonly used in news reports.
```

Comment:
They are "must" and "can", because the fist is compulsory and the second is optional.